test-interface/src/main/scala/scala/scalanative/testinterface/TestMain.scala

```diff
@@ -38,136 +22,17 @@ object TestMain {
 
   /** Main method of the test runner. */
   def main(args: Array[String]): Unit = {
-    if (args.length < 2) {
+    if (args.length < 1) {
```

Comment:
I suggest ` if (args.length != 1)` and changing 2 lines below:
`one argument expected`.  The usage(), with its more informative text
will have already been given, so I am not suggesting my usual:
"5 arguments found, 1 expected" or "Expected 1 argument, none found".